models/ActiveRecord.php

```diff
@@ -189,7 +189,7 @@ public static function get(string $order = 'ASC', int $limit = 0)
     {
         $query = "SELECT * FROM " . static::$tableName . " ORDER BY " . static::$primaryKey . " $order";
         
-        if ($limit === 0) $query .= " LIMIT $limit";
+        if ($limit == 0) $query .= " LIMIT $limit";
```

Comment:
Aqui hay que hacer que sea distinta de 0, es decir, si hay un limite se agrega.
```suggestion
        if ($limit !== 0) $query .= " LIMIT $limit";
```